angular/projects/catalyst-demo/src/app/app.component.ts

```diff
@@ -40,6 +42,93 @@ export class AppComponent implements OnInit {
     render: (country: Country) => ({ label: country.country })
   };
 
+  fields: FormlyFieldConfig[] = [
+    {
+      key: 'catCheckbox',
+      type: CatCheckboxFieldType,
+      props: {
+        label: 'Custom Cat Checkbox'
+      },
+    },
+    {
+      key: 'catToggle',
+      type: CatToggleFieldType,
+      props: {
+        label: 'Custom Cat Toggle'
+      },
+    },
+    {
+      key: 'catRadio',
+      type: CatRadioFieldType,
+      props: {
+        label: 'Custom Cat Radio'
+      },
+    },
+    {
+      key: 'catInput',
+      type: CatInputFieldType,
+      defaultValue: 'type here',
+      props: {
+        label: 'Custom Cat Input',
+        clearable: true,
+        required: true
+      }
+    },
+    {
+      key: 'catTextarea',
+      type: CatTextareaFieldType,
+      defaultValue: 'type inside textarea',
+      props: {
+        label: 'Custom Cat Input',
+        rows: 1
+      },
+    },
+    {
+      key: 'catRadioGroup',
+      type: CatRadioGroupFieldType,
+      defaultValue: 'two',
+      props: {
+        a11yLabel: 'Radio Group 1',
+        labelLeft: false,
+        options: [
+          {
+            label: 'Option 1 - Group 1',
+            value: 'one'
+          },
+          {
+            label: 'Option 2 - Group 1',
+            value: 'two'
+          },
+          {
+            label: 'Option 3 - Group 1',
+            value: 'three',
+            checked: true,
+          },
+        ]
+      }
+    },
+    {
+      key: 'catDatepicker',
+      type: CatDatepickerFieldType,
+      defaultValue: '12.12.2022',
```

Comment:
does this work? I thought I built it in a way where the date is expected as ISO string. I'd change it to an ISO string

Reply:
Yes, it works this way